src/codeu/chat/client/simplegui/MessagePanel.java

```diff
@@ -121,7 +121,6 @@ private void initialize() {
     final JButton addButton = new JButton("Send Message");
     buttonPanel.add(textField); //Adds the message box before the "Send Message" button
     buttonPanel.add(addButton);
-    
```

Comment:
nit: this file does not need to be in this PR. There is only a blank line deleted.